frontend/test/specs/event/eventDialogCtrlSpec.js

```diff
@@ -315,17 +308,190 @@
         });
       }); 
 
-      describe('isEventOutDate', function() {
-        it('should return true', function() {
+      describe('isEventOutDate', () => {
+        it('should return true', () => {
           controller.event.end_time.setFullYear(2000);
           expect(controller.isEventOutdated()).toBe(true);
         });
 
-        it('should return false', function() {
+        it('should return false', () => {
           controller.event.end_time.setFullYear(3100);
           expect(controller.isEventOutdated()).toBe(false);
         });
       });
     });
+
+    describe('cancelCreation()', () => {
+      it('should call state.go if is mobile screen', () => {
+        spyOn(Utils, 'isMobileScreen').and.returnValue(true);
+        spyOn(state, 'go');
+        controller.cancelCreation();
+        expect(state.go).toHaveBeenCalled();
+      });
+
+      it('should call mdDialog.hide if is not mobile screen', () => {
+        spyOn(Utils, 'isMobileScreen').and.returnValue(false);
+        spyOn(mdDialog, 'hide');
+        controller.cancelCreation();
+        expect(mdDialog.hide).toHaveBeenCalled();
+      });
+    });
+
+    describe('lastStep()', () => {
+      it('should get the step 4 if is mobile screen', () => {
+        spyOn(Utils, 'isMobileScreen').and.returnValue(true);
+        spyOn(controller, 'getStep');
+        controller.lastStep();
+        expect(controller.getStep).toHaveBeenCalledWith(4);
+      });
+
+      it('should get the step 3 if is not mobile screen', () => {
+        spyOn(Utils, 'isMobileScreen').and.returnValue(false);
+        spyOn(controller, 'getStep');
+        controller.lastStep();
+        expect(controller.getStep).toHaveBeenCalledWith(3);
+      });
+    });
+
+    describe('getStreetPlaceholder()', () => {
+      const street = "Rua";
+      it('should return "Rua" if another country is selected', () => {
+        controller.isAnotherCountry = true;
+        expect(controller.getStreetPlaceholder()).toEqual(street);
+      });
+
+      it('should return "Rua *" if another country is not selected', () => {
+        controller.isAnotherCountry = false;
+        expect(controller.getStreetPlaceholder()).toEqual(street + " *");
+      });
+    });
+
+    describe('_loadStateParams()', () => {
+      beforeEach(() => {
+        state.params.event = event;
+        state.params.events = [event];
+        state.params.isEditing = true;
+        controller.event = null;
```

Comment:
i do prefer to use undefined.
and for verification you could use toBe(undefined)